inotify.go

```diff
@@ -36,7 +37,7 @@ type Watcher struct {
 // NewWatcher establishes a new watcher with the underlying OS and begins waiting for events.
 func NewWatcher() (*Watcher, error) {
 	// Create inotify fd
-	fd, errno := unix.InotifyInit()
+	fd, errno := unix.InotifyInit1(syscall.IN_CLOEXEC)
```

Comment:
Hoping the Go team will help us out with that, but I'm not sure when they'll have time. https://github.com/golang/go/issues/17312